ostelco-ios-client/Coordinators/OnboardingCoordinator.swift

```diff
@@ -250,26 +126,26 @@ extension OnboardingCoordinator: LoginDelegate {
     
     func signedIn(controller: UIViewController, authCode: String, contactEmail: String?) {
         let spinnerView = controller.showSpinner()
-
+        
```

Comment:
Are these tabs?  Is there a policy on tabs v.s. spaces for swift code?

Reply:
We use what the editor does by default.